airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -161,6 +161,23 @@ def get_client(self, project_id: Optional[str] = None, location: Optional[str] =
             credentials=self._get_credentials(),
         )
 
+    def get_uri(self) -> str:
+        """Override DbApiHook get_uri method for get_sqlalchemy_engine()"""
+        return "bigquery://"
```

Comment:
This is not a standard driver. We need to install the [`sqlalchemy-bigquery`](https://github.com/googleapis/python-bigquery-sqlalchemy) package to use it. Can you add this as an Google provider dependency?

Reply:
sure, on it

Reply:
@mik-laj forgive my ignorance, how can I add this package into google provider?
[here](https://github.com/apache/airflow/blob/main/airflow/providers/google/provider.yaml#L809)?

Reply:
You need to update setup.py https://github.com/apache/airflow/blob/d353f023ff8856c00b9f054526cb2e40ff0116ae/setup.py#L294-L346

Reply:
oh I see, thx~

Reply:
done~